### PR TITLE
nginx config generator

### PR DESCRIPTION
**Pull request by javierg**
doing it for da #18

### CODE REVIEW

File: config/vhost.conf.erb [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  server_name www.rt.dev *.rt.dev;
+
+  root /Users/guerra/rt/public;
+  passenger_enabled on;
+
+  rack_env development;
+  passenger_friendly_error_pages on;
+
+  access_log /Users/guerra/rt/nginx_access.log;
+  access_log /Users/guerra/rt/nginx_error.log;
```
Comment (EnriqueVidal) [inline config/vhost.conf.erb]: :bomb:

**Comment by EnriqueVidal** [comment]
You left out the `gzip_static` part for `public/assets`

**Comment by javierg** [comment]
Yeah, didn't took it in account

do we urgently need that?  we can add that in another commit

File: config/vhost.conf.erb (hunk 1 above)
Comment (javierg) [inline config/vhost.conf.erb]: Dam!! I was sleeapy... sorry let me fix that

**Comment by EnriqueVidal** [comment]
@javierg  for production we do.

**Pull request merged**